CHANGES

```diff
@@ -17,6 +17,8 @@ Schema Changes
 - Added ``Release.organization`` column.
 - Added ``ReleaseProject`` model.
 - Added ``ReleaseCommit.organization_id`` column.
+- Added ``ReleaseFile.organization`` column.
```

Comment:
🤔 In an ideal world, probably. But I don't have a good answer, I didn't start this pattern. :) It's fine as-is, hence why I approved it.